test/AutoDiff/Serialization/derivative_attr.swift

```diff
@@ -56,8 +56,11 @@ extension S {
     (self, { $0 })
   }
 
+  // Note: qualified name base types are not yet serialized and are not printed
+  // when round-tripping.
+
   // CHECK: @derivative(of: instanceMethod, wrt: (self, x))
-  @derivative(of: instanceMethod, wrt: (self, x))
+  @derivative(of: S.instanceMethod, wrt: (self, x))
```

Comment:
Note: qualified name base types are not yet serialized for `@derivative` or `@transpose` attributes.

Example: `@derivative(of: S.instanceMethod)` is serialized and subsequently printed as `@derivative(of: instanceMethod)`.

This isn't yet an issue in practice, because `@derivative` and `@transpose` declarations are designed to be declared in the same type context as their referenced original declaration. We can tackle it if it becomes a problem.